components/speedreader/buildflags.gni

```diff
@@ -1,10 +1,3 @@
-import("//brave/build/config.gni")
-import("//build/config/features.gni")
-
 declare_args() {
   enable_speedreader = !is_android
-  enable_speedreader_feature = brave_channel == "nightly"
```

Comment:
To avoid an import loop error when importing components/speedreader/buildflags.gni in //brave/browser/sources.gni, I make a try to move the declaration of **enable_speedreader_feature** from here to //brave/components/speedreader/BUILD.gn. Please let me know if this movement is not fine.